### DISCUSSION TITLE
🧠 Galaxy brain 🧠

### DISCUSSION THREAD

**Discussion by LarsPeterson** [Schweinepriester/github-profile-achievements#579] [Q&A] [answered]
I figured it out...... 🤦‍♂️

**Comment by RealMrHex** [comment, marked as answer]
@LarsPeterson yup me too 😆

**Reply by LarsPeterson** [reply]
Thanks bro

**Reply by RealMrHex** [reply]
> Thanks bro

You are very welcome!
I just need one more plz :)

**Comment by LarsPeterson** [comment]
![image](https://user-images.githubusercontent.com/25396567/213310242-fd0215e7-afb1-4194-a052-f7f1e3cb567f.png)

@RealMrHex can you recreate a Q&A discussion?

**Reply by RealMrHex** [reply]
Need another?

**Reply by LarsPeterson** [reply]
Nope all good
